packages/date/package.json

```diff
@@ -31,7 +31,7 @@
     "yup": "^0.30.0"
   },
   "peerDependencies": {
-    "@availity/form": "^0.1.0",
+    "@availity/form": "^0.1.0 || ^0.5.34",
```

Comment:
Can you add the in between versions too? like 0.2, 0.3, and 0.4? I would also change `^0.5.34` to `^0.5.0`